packages/@ionic/cli/src/commands/start.ts

```diff
@@ -539,9 +539,8 @@ Use the ${input('--type')} option to start projects using older versions of Ioni
     this.env.shell.alterPath = p => prependNodeModulesBinToPath(projectDir, p);
 
     if (!this.schema.cloned) {
-      if (this.schema.type === 'react') {
-        options['capacitor'] = true;
-      }
+      // Default to capacitor always
+      options['capacitor'] = true;
```

Comment:
We probably should not do this for Angular yet until the starters are updated to remove the Cordova plugins and related code.

We will also need to improve the Cordova integration to install those plugins and advise the dev how to use the splash screen.

Reply:
Agree. Will revert this and add a check for vue instead.